src/Misc/jordan.jl

```diff
@@ -812,12 +812,12 @@ of a key and a value, then `v` is the eigenspace of `L[i]` w.r.t. the eigenvalue
 If side is `:right`, the right eigenspaces are computed, if it is `:left` then the
 left eigenspaces are computed.
 """
-function common_eigenspaces(L::Vector{<: MatElem{T}}; side::Symbol = :right) where T <: FieldElem
-  eigs = [ eigenspaces(M, side = side) for M in L ]
+function common_eigenspaces(L::Vector{<: MatElem{T}}; side::Symbol = :left) where T <: FieldElem
+  eigs = [ eigenspaces(M; side = :left) for M in L ]
```

Comment:
```suggestion
  eigs = [ eigenspaces(M; side = side) for M in L ]
```
I don't think anything should be changed here?